src/test/validation_chainstatemanager_tests.cpp

```diff
@@ -659,8 +660,16 @@ BOOST_FIXTURE_TEST_CASE(chainstatemanager_snapshot_completion_hash_mismatch, Sna
     fs::path snapshot_chainstate_dir = gArgs.GetDataDirNet() / "chainstate_snapshot";
     BOOST_CHECK(fs::exists(snapshot_chainstate_dir));
 
-    res = WITH_LOCK(::cs_main, return chainman.MaybeCompleteSnapshotValidation());
-    BOOST_CHECK_EQUAL(res, SnapshotCompletionResult::HASH_MISMATCH);
+    {
+        bool error_printed = false;
+        DebugLogHelper log_helper("Bitcoin Core failed to validate the -assumeutxo snapshot state", [&](const std::string* s) {
+            if (s) error_printed = true;
+            return false;
+        });
```

Comment:
Didn't know that the macro existed. Thanks.

> Finally, it may be good to add steps to reproduce/test for reviewers. Otherwise it is unclear what your goal is. Reading the description you are trying to avoid that some error string is output?

Yeah. The `chainstatemanager_snapshot_completion_hash_mismatch` unit test manually triggers a fatal error, and the test framework by default prints the error message to the console ([this one](https://github.com/bitcoin/bitcoin/blob/79e8247ddb166f9b980f40249b7372a502402a4d/src/validation.cpp#L5406)).

So, just by running the unit test suite, you will see a long AssumeUTXO fatal error even when all tests passed successfully.